pages/01.home/01.contributing/docs.en.md

```diff
@@ -158,7 +158,7 @@ The **`README.md`** file serves as the introduction and description of this repo
 
 The **`SUMMARY.md`** file defines the menu of the documentation. If you add a new page to the documentation, you'll have to also add a new line there defining the title and the link to the file (formatted like the existing menu items).
 
-The folders in this repository are grouped together by topic. For example, within the *asset* folder, you'll see it has its own **`README.md`** file (becomes `./index.html`) which contains the primary description of the Asset menu; the *manage_assets.md* file is a subitem; the *media* subfolder contains all the images used in the *.md* files.
+The folders in this repository are grouped together by topic. For example, within the *asset* folder, you'll see it has its own **`doc.en.md`** file (becomes `./index.html`) which contains the primary description of the Asset menu; the *manage_assets.md* the file is a subitem; the *media* subfolder contains all the images used in the *.md* files.
```

Comment:
Thanks for the tweaks, just a small bit that needs updating: it actually becomes /assets (preceded by the path to the nested location in the menu) and the manage assets file is also docs.en.md and becomes /managing-assets - see https://docs.mautic.org/en/components/assets/.

Here is the [documentation from Grav](https://learn.getgrav.org/16/content/content-pages), the flat file CMS we are using, and here are some docs on the [theme we are using & how it works](https://learn.hibbittsdesign.org/learn2withgitsync) - these might be helpful in understanding how it all works together!

Reply:
_The folders in this repository are grouped together by topic. For example, within the *asset* folder, you'll see it has its own **`doc.en.md`** file (becomes `/assets`) which contains the primary description of the Asset menu; the *manage_assets.md* the file is a subitem it is also has a **`doc.en.md`** file (becomes `/managing-assets`); the *media* subfolder contains all the images used in the *.md* files._


@RCheesley I don't seem to fully understand how this works I am new to Grav but I am going through the links you gave meanwhile i have made some tweaks

Reply:
@RCheesley I will look into this later today